### PR TITLE
Fix `brew tests` leaving last file in trash.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

Fixes https://github.com/Homebrew/brew/issues/2974#issuecomment-333296864.

### CODE REVIEW

File: Library/Homebrew/cask/lib/hbc/artifact/abstract_uninstall.rb [hunk 1]
```diff
@@ -267,14 +267,17 @@ def trash_paths(*paths, command: nil, **_)
...
                 end if
               end repeat
 
               return output
             end tell
           end run
         EOS
+
+        # Remove AppleScript's automatic newline.
+        result.tap { |r| r.stdout.sub!(/\n$/, "") }
```
Comment (ilovezfs) [inline Library/Homebrew/cask/lib/hbc/artifact/abstract_uninstall.rb]: `chomp!` doesn't work?

**Comment by MikeMcQuaid** [comment]
@reitermarkus This is still not fixing https://github.com/Homebrew/brew/issues/2974. Please guard this test for `ENV["CI"]` or something similar or I'm just going to remove the test entirely within a few days.

**Comment by reitermarkus** [comment]
Guarded it with `HOMEBREW_TESTS_COVERAGE`.

File: Library/Homebrew/cask/lib/hbc/artifact/abstract_uninstall.rb (hunk 1 above)
Comment (reitermarkus) [reply Library/Homebrew/cask/lib/hbc/artifact/abstract_uninstall.rb]: `chomp` doesn't work on the `\r\n` edge case.

**Comment by MikeMcQuaid** [comment]
Thanks @reitermarkus!

**Pull request merged**